function.js

```diff
@@ -0,0 +1,154 @@
+// // Перепишите функцию, чтобы она делала то же самое, но без if, в одну строку.
+// // Сделайте два варианта функции checkAge:
+// // Используя оператор ?
+// // Используя оператор ||
+
+let checkAge(age) {
+    return (age >= 18) ? true : confirm("Родители разрешили?")
+};
+
+let checkAge(age) {
+    return (age >= 18) || confirm("Родители разрешили?")
+};
+
+let checkAge(age) ? () => true : () confirm("Родители разрешили?");
+
+
+// // Напишите функцию min(a,b), которая возвращает меньшее из чисел a и b.
+function min(a, b) {
+    if (a < b) {
+        return a;
+    } else {
+        return b;
+    }
+};
+
+// Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; -1, если первое число меньше, чем второе, и 0, если числа равны.
+
+function getVar(a, b) {
+    if (a > b) {
+        return 1;
+    } else if (b > a) {
+        return -1;
+    } else {
+        return 0;
+    };
```

Comment:
You do it need thıs else 

remove unnecessary code


this part of code wıll be executed at any case if two previous conditions will not pass

Reply:
fixed